dim-testsuite/t/zone-import-3.t

```diff
@@ -5,28 +5,27 @@ WARNING - Primary NS for this Domain is now localhost.
 $ cat <<EOF | ndcli import zone example.com view us
 @               IN      SOA     ins01.internal.test. dnsadmin.company.com. 2005110981 600 180 604800 3600
 
-*.app1.devqa.diy                IN CNAME        vm3336.development.lan. ; 2012-12-10 15:37:30 : 
+*.app1.devqa.diy                IN CNAME        vm3336.development.lan. ; 2012-12-10 15:37:30 :
 *.jimdo-dev16.diy               IN CNAME        jimdo-dev16.diy ; 2012-06-13 11:03:04 : ITOCIS-7783
 EOF
 INFO - Creating zone example.com view us
-RECORD - example.com. IN SOA ins01.internal.test. dnsadmin.company.com. 2005110981 600 180 604800 3600
+RECORD - example.com. 3600 IN SOA ins01.internal.test. dnsadmin.company.com. 2005110981 600 180 604800 3600
 INFO - Creating RR @ SOA ins01.internal.test. dnsadmin.company.com. 2005110981 600 180 604800 3600 in zone example.com view us
-RECORD - *.app1.devqa.diy.example.com. IN CNAME vm3336.development.lan. ; 2012-12-10 15:37:30 :
-INFO - Creating RR *.app1.devqa.diy CNAME vm3336.development.lan. comment 2012-12-10 15:37:30 : in zone example.com view us
-WARNING - jimdo-dev16.diy.example.com. does not exist.
-RECORD - *.jimdo-dev16.diy.example.com. IN CNAME jimdo-dev16.diy ; 2012-06-13 11:03:04 : ITOCIS-7783
-INFO - Creating RR *.jimdo-dev16.diy CNAME jimdo-dev16.diy comment 2012-06-13 11:03:04 : ITOCIS-7783 in zone example.com view us
+RECORD - *.app1.devqa.diy.example.com. 3600 IN CNAME vm3336.development.lan.
+INFO - Creating RR *.app1.devqa.diy CNAME vm3336.development.lan. in zone example.com view us
 WARNING - vm3336.development.lan. does not exist.
+RECORD - *.jimdo-dev16.diy.example.com. 3600 IN CNAME jimdo-dev16.diy.example.com.
+INFO - Creating RR *.jimdo-dev16.diy CNAME jimdo-dev16.diy.example.com. in zone example.com view us
+WARNING - jimdo-dev16.diy.example.com. does not exist.
```

Comment:
Comments are currently not imported, as the dns library removed the functionality.
But is it correct that the zone is added to every log entry?

Reply:
CNAMEs now look to be imported as absolut. 
e.g.
`Creating RR *.jimdo-dev16.diy CNAME jimdo-dev16.diy.example.com. in zone example.com view us`
instead of being relative
`Creating RR *.jimdo-dev16.diy CNAME jimdo-dev16.diy in zone example.com view us`

This is not changing functionality, so it is ok.